tap/tcp_streams_map.go

```diff
@@ -67,6 +66,9 @@ func (streamMap *tcpStreamMap) CloseTimedoutTcpStreamChannels() {
 						diagnose.AppStats.DroppedTcpStreams, runtime.NumGoroutine(), tcpStreamChannelTimeoutMs/time.Millisecond)
 				}
 			}
+			if stream.client.doneReading && stream.server.doneReading {
```

Comment:
Function name `CloseTimedoutTcpStreamChannels` doesn't suite this function anymore because it also closes done streams, not only timed-out streams.
Rename. I suggest `RunTcpStreamChannelsCleanup`.